testnets/sgenetworktestnet/assetlist.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_id": "sge-network-3",
+  "assets": [
+    {
+      "description": "The SGE token is primarily a governance token for the SGE Network blockchain.",
+      "denom_units": [
+        {
+          "denom": "usge",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "usge",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "usge",
+      "name": "SGE Network",
+      "display": "sge",
+      "symbol": "SGE",
+      "logo_URIs": {
+        "png": "https://sgenetwork.io/images/sgenetwork-logo.png",
+        "svg": "https://sgenetwork.io/images/logo-SGE.svg"
+      }
+    },
+    {
+      "description": "The SGE token is primarily a governance token for the SGE Network blockchain.",
+      "denom_units": [
+        {
+          "denom":"ibc/CFDF212C4BE8349AEDF9DE5006F394717C7E9E814742E1D6DE95AE288F58054C",
+          "exponent": 0,
+          "aliases": [
+            "usge"
+          ]
+        },
+        {
+          "denom": "usge",
+          "exponent": 6
+        }
+      ],
+      "base":"ibc/CFDF212C4BE8349AEDF9DE5006F394717C7E9E814742E1D6DE95AE288F58054C",
+      "name": "SGE Network",
+      "display": "sge",
+      "symbol": "SGE",
+      "ibc": {
```

Comment:
we don't use this property anymore (we use 'traces'). What is this ibc transfer describing?

Reply:
@JeremyParish69
Those are added thinking, transferring tokens from keplr to osmosis or osmosis to keplr?  - is it wrong? if yes,  can you please share more info about it and what should we do?

Reply:
oh, so this is describing the transition from sgetestnet to osmosistestnet? If that's the case, it should not be listed here. I'd remove the ibc asset entirely.